references/organization/_06-Project-Plan-and-Workflow.md

```diff
@@ -0,0 +1,64 @@
+
+##### Concrete steps: 
+
+Functions <br>
+-	A class with functions to estimate factor models. Start with PPCA as in the deconfounder’s tutorial and expand as needed. <br>
+-	A function to perform posterior predictive checks on the factor model to assess its predictive accuracy. <br>
+
+Workflow <br>
+-	Start with a hypothesized general causal graph. For the simulated example, we’ll start by assuming the true causal graph used to generate the data, where we highlight the confounder as being latent/unobserved. <br>
+-	Then, come up with a “utility level” causal graph, where we come up with a causal graph for each utility equation since the confounding is at the level of the utilities in a MNL context. These causal graphs would inform the utility specifications, and will determine which utilities do and do not require estimating a latent confounder. We discussed this briefly before, but are we on the same page? <br>
+-	Based on the above specifications, estimate the required latent confounders using a factor model, start with PPCA. <br>
+-	Perform posterior predictive checks on the estimated factor model and assess goodness of fit. Since we’re doing this on a simulated dataset, then we know in advance how many latent confounders there are. We could potentially try controlling for more confounders and see how that affects the results (in other words, see how sensitive our analysis is to the number of confounders used). <br>
+-	Once the checks pass, include the latent confounder in the utility equations, and estimate a MNL. <br>
+-	Repeat the process n times, and calculate the percentage of those times where the estimated parameters’ confidence intervals contain the true parameter. <br>
+Finally, we can repeat the above process, but now on the full, real bike dataset. The workflow would be the same as above, with the addition of one step: <br>
+-	After hypothesizing on a causal graph, we need to do some conditional independence testing to falsify the graph by making sure nothing clearly violates conditional independence tests. The caveat here is that there don’t seem to be one way to perform conditional independence testing – there are many ways proposed, but I’m not sure if the problem itself is solved yet. We can try one way, and point out that this is an active area of research and that there exists other ways to do it. The important take away is that the causal graph should not clearly violate a given test, in which case something is obviously wrong. <br>
```

Comment:
I think we should call out two explicit demonstrations here:
1. a test of conditional independence amongst the observable variables in the graph
2. a test of the implied relationships due to the latent variables and their hypothesized structure

For the first one, you've got a demonstration in #37 .
For the second, I will brainstorm!

Either way, everything will be couched exactly as you said: many ways proposed, but no matter which we use, we should not clearly violate any given test.

Reply:
@timothyb0912  I don't follow you on the second test. We can discuss more later, but I'll leave it out for now.

Reply:
Yeah no worries. I'm happy to discuss if you want but I second your suggestion of leaving it out for now as these sort of tests look difficult/non-trivial to implement based on my readings!